inventory/rh-sso/group_vars/rh-sso-hosts.yml

```diff
@@ -25,3 +25,20 @@ realms:
     id: simple-realm
     displayName: "Simple Times"
     enabled: false
+
+id_providers:
+  - alias: google
+    displayName: google-test-1
+    providerId: google
+    enabled: true
+    internalId: google
+    linkOnly: false
+    firstBrokerLoginFlowAlias: first broker login
+    postBrokerLoginFlowAlias:
```

Comment:
Can we add a dummy value here?

Reply:
Absolutely. I am adding the first default from the list and defaulting it to an empty value when no additional authentication flow is required.